### DISCUSSION TITLE
文件抓取名称不全

### DISCUSSION THREAD

**Discussion by LHT-Liu** [zotero-chinese/wiki#111] [求助 / Q&A] [answered]
![image](https://github.com/zotero-chinese/zotero-plugins/assets/64388319/0c708630-3a97-4a53-ac2c-e0c237171e9b)
请问这个我目前是右击pdf文件，Rename file from parent matadata，但是总是显示不全，这个该怎么解决？感谢

**Comment by northword** [comment]
Zotero 7 参考 https://www.zotero.org/support/file_renaming `truncate`

---

此 Issue 将被转移至 discussion

**Reply by LHT-Liu** [reply]
zotero 6 请问怎么处理呢？

**Reply by northword** [reply]
我已经七八个月没有用过 Zotero 6 了，记不清了。

Z 6 也有相关的首选项，只不过是隐藏的，需要自己寻找修改，且没有 7 完善。

Z 列表中父级条目已经显示了文献标题，所以在附件名体现文件标题是不必要的（注意区分附件名和文件名，z6区分尚不明确，z7 已经明确区分了这一点），附件名应该体现的是“全文pdf”、“支持材料”、“笔记”等。

另可以通过 zotfile 实现重命名（不推荐，zotfile 已经停止维护）

**Reply by LHT-Liu** [reply]
请问z7父级标题不体现文献名字，那一般怎么直观体现文章的名字等信息，直接看右侧吗？可以来截图介绍一下吗？

**Reply by northword** [reply, marked as answer]
![image](https://github.com/zotero-chinese/wiki/assets/44738481/de0ee1ff-c329-4915-b076-eeca4a7b3c24)

**Reply by LHT-Liu** [reply]
兄弟秒回呀，感谢，我明白你的意思了

**Reply by northword** [reply]
给哈哈，刚好在电脑前。

附件名和文件名的区别，文件名是PDF实际的名称，附件名是在 Zotero 里显示的标题
![image](https://github.com/zotero-chinese/wiki/assets/44738481/1ecdefe5-eb8f-44fe-826b-488751d5833c)

**Reply by LHT-Liu** [reply]
好的，感谢您
